connectors/sources/github.py

```diff
@@ -1436,6 +1447,9 @@ async def _fetch_pull_requests(
         response_key=(REPOSITORY_OBJECT, "pullRequests"),
         filter_query=None,
     ):
+        self._logger.info(
```

Comment:
What's a `response_key`? Is it a token?

Reply:
I think it's which fields from the response object we want back? I dont' really follow the logic and am not familiar with the graphql github API, but I wanted to log everything relevant for us to be able to follow the logic flow and re-create the request if necessary (if DEBUG logs aren't enabled)